format/riff/testdata/dolby_metadata.fqtest

```diff
@@ -0,0 +1,192 @@
+      |00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f|0123456789abcdef|.{}: ./format/riff/testdata/dolby_metadata.wav (wav) 0x0-0x2aec (10988)
```

Comment:
There should be a starting `$ fq dv dolby_metadata.wav` line etc here. Then you can use `-update`when you run the tests to automatically update with expected output